2021/2021-09-07.md

```diff
@@ -0,0 +1,94 @@
+# Kit Team Meeting 2021-09-07
+
+## Attendees
+
+### Present
+
+- Dan Trickey
+- Karina Kwiatek
+- Tyler Ward
+- Jake Howard
+- Tom Wheal
+- Will Barber
+- Andy Barrett-Sprot
+
+
+## Agenda
+
+1. Structure of Kit Team and priorities for the next year
+2. Recovery of missing kit from SR2017, 2019 and 2020.
+3. Asset Audit
+4. Feasibility of shipping physical kit for SR2022
+5. Kit Software progress and plan (ex-Brain Team work)
+6. Documentation (i.e deprecation of Kitbook)
+7. Regular meeting slot going forward
+8. Blog Posts for website
+
+## Topics
+
+- Recovery of missing kit from SR2017, 2019 and 2020.
+    - 4 / 6 kits from schools that returned are in Tom's Garage
+    - Been unable to reach 14 schools - staff retiring
+    - 18 kits are being kept by school. Safe to assume that they're good or can send spares
+    - 8 kits in Tom's garage
+    - Morse suggested sending signed letters to headmaster
+    - Ask teams holding on to kits to audit them?
+    - No hard number today
+
+- Feasibility of shipping
+    - We could drop vision. It would not be a huge amount of work
+    - We've been able to give an alternative instead. Without vision how would they navigate?
```

Comment:
Is the tense here correct? From context it feels like this maybe should say "We'd need to be able to give an alternative"?
Alternatively, if this is correct, what alternative is it referring to?